ush/get_data_files.py

```diff
@@ -460,9 +468,17 @@ def get_hpss_data(hpss_job_filename, save_data_dir, save_data_file,
                   +'--job-name='+hpss_job_name+' '+hpss_job_filename)
         job_check_cmd = ('squeue -u '+os.environ['USER']+' -n '
                          +hpss_job_name+' -t R,PD -h | wc -l')
-    elif machine in ['ORION', 'S4', 'HERCULES', 'GAEAC5', 'GAEAC6']:
+    elif machine in ['GAEAC6']:
+        os.system('sbatch --ntasks=1 --time='
+                  +walltime.strftime('%H:%M:%S')+' --clusters='+CLUSTERS_DTN+' '  #ES
+                  +' --partition='+PARTITION_DTN+' '                              #dtn_f5_f6 '
+                  +'--account='+ACCOUNT+' --output='+hpss_job_output+' '
+                  +'--job-name='+hpss_job_name+' '+hpss_job_filename)
+        job_check_cmd = ('squeue -u '+os.environ['USER']+' -n '
+                         +hpss_job_name+' -t R,PD -h | wc -l')
+    elif machine in ['ORION', 'S4', 'HERCULES', 'GAEAC5']:    #, 'GAEAC6']:
         print("ERROR: No HPSS access from "+machine)
-    if machine not in ['ORION', 'S4', 'HERCULES', 'GAEAC5', 'GAEAC6']:
+    if machine not in ['ORION', 'S4', 'HERCULES', 'GAEAC5']:  #, 'GAEAC6']:
```

Comment:
No need to keep the GAEAC6 comment.  This can be tracked in history if needed.

```suggestion
    if machine not in ['ORION', 'S4', 'HERCULES', 'GAEAC5']:
```

Reply:
Thank you @DavidHuber-NOAA. Should I go ahead and add the changes for C5 as well? I will run a quick test to make sure, but I expect them to work?

Reply:
No need to extend support to C5.  As far as I know, no one is utilizing that cluster nor is anyone supporting it.  But if you would like to make use of it, please feel free to make the updates.